### PR TITLE
Add test for Ground truth Job list

### PR DESCRIPTION
**Pull request by klakhov**
### Motivation and context

Test for bug which was fixed in #7441

### Checklist

- [x] I submit my changes into the `develop` branch
- ~~[ ] I have created a changelog fragment ~~
- ~~[ ] I have updated the documentation accordingly~~
- [x] I have added tests to cover my changes
- ~~[ ] I have linked related issues (see [GitHub docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))~~
- ~~[ ] I have increased versions of npm packages if it is necessary
  ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
  [cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning),
  [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and
  [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))~~

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

### CODE REVIEW

File: tests/cypress/e2e/features/ground_truth_jobs.js [hunk 1]
```diff
@@ -397,4 +415,63 @@ context('Ground truth jobs', () => {
             cy.checkFrameNum(groundTruthFrames[3]);
         });
     });
+
+    describe('Testing case ground truth job list', () => {
+        const imagesCount = 60;
```
Comment (bsekachev) [inline tests/cypress/e2e/features/ground_truth_jobs.js:420]: I would propose 20 images with segment size 1

File: tests/cypress/e2e/features/ground_truth_jobs.js [hunk 2]
```diff
@@ -397,4 +415,63 @@ context('Ground truth jobs', () => {
...
+            cy.saveJob();
+
+            cy.interactMenu('Finish the job');
+            cy.get('.cvat-modal-content-finish-job').within(() => {
+                cy.contains('button', 'Continue').click();
+            });
+
+            createTaskQualityReport(taskID);
+
+            for (let i = 0; i < 3; i++) {
```
Comment (bsekachev) [inline tests/cypress/e2e/features/ground_truth_jobs.js:466]: 3 is a magic number, it is not obvious that it result of dividing number of images to segment size

File: tests/cypress/e2e/features/ground_truth_jobs.js [hunk 3]
```diff
@@ -397,4 +415,63 @@ context('Ground truth jobs', () => {
...
+
+        before(() => {
+            cy.visit('/tasks');
+            cy.imageGenerator(imagesFolder, imageFileName, width, height, color, posX, posY, labelName, imagesCount);
+            cy.createZipArchive(directoryToArchive, archivePath);
+            cy.createAnnotationTask(
+                taskName, labelName, attrName,
+                textDefaultValue, archiveName, false,
+                { multiJobs: true, segmentSize: 2 },
+            );
```
Comment (bsekachev) [inline tests/cypress/e2e/features/ground_truth_jobs.js:440]: if possible I would propose to use headless creating

File: tests/cypress/e2e/features/ground_truth_jobs.js [hunk 4]
```diff
@@ -397,4 +415,63 @@ context('Ground truth jobs', () => {
...
+            cy.get('.cvat-modal-content-finish-job').within(() => {
+                cy.contains('button', 'Continue').click();
+            });
+
+            createTaskQualityReport(taskID);
+
+            for (let i = 0; i < 3; i++) {
+                cy.get('.cvat-task-jobs-table-row').each((row) => {
+                    cy.get(row).within(() => {
+                        cy.contains('N/A').should('not.exist');
```
Comment (bsekachev) [inline tests/cypress/e2e/features/ground_truth_jobs.js:469]: .should('not.include.text', 'hello')

File: tests/cypress/e2e/features/ground_truth_jobs.js [hunk 5]
```diff
@@ -397,4 +415,63 @@ context('Ground truth jobs', () => {
...
+                quantity: 100,
+            });
+
+            cy.createRectangle(rectangleTrack);
+            cy.saveJob();
+
+            cy.interactMenu('Finish the job');
+            cy.get('.cvat-modal-content-finish-job').within(() => {
+                cy.contains('button', 'Continue').click();
+            });
```
Comment (bsekachev) [inline tests/cypress/e2e/features/ground_truth_jobs.js:462]: You may just push these as headless requests after the task is created

File: tests/cypress/e2e/features/ground_truth_jobs.js (hunk 3 above)
Comment (klakhov) [reply tests/cypress/e2e/features/ground_truth_jobs.js]: As I remember there was a problem with attaching files to cypress requests, so for now ill stick to manual task creation

**Review by bsekachev — APPROVED** [review]

**Pull request merged**